pkg/machines/components/storagePools/storagePoolVolumesTab.jsx

```diff
@@ -80,19 +76,18 @@ export class StoragePoolVolumesTab extends React.Component {
         const storagePoolIdPrefix = storagePoolId(storagePool.name, storagePool.connectionName);
         const volumes = this.state.rows;
         const isVolumeUsed = getStorageVolumesUsage(vms, storagePool);
-        const columnTitles = [
-            _("Name"), _("Used by"), _("Size"),
-            {
-                title: <div className='table-actions pull-right'>
-                    <StorageVolumeDelete key='volume-delete-action'
-                            storagePool={storagePool}
-                            isVolumeUsed={isVolumeUsed}
-                            volumes={volumes.filter(row => row.selected)}
-                            deleteErrorHandler={this.deleteErrorHandler} />
-                    <StorageVolumeCreate key='volume-create-action'
-                            storagePool={storagePool} />
-                </div>
-            }];
+        const columnTitles = [_("Name"), _("Used by"), _("Size")];
+        const actions = (
+            <div className='table-actions pull-right'>
```

Comment:
Is the pull right class actually needed still, since the h3 element next to it has flex set to auto?
I tried removing it and I couldn't spot any difference. I know @garrett hates these, so might be good to just clean it out in the process.

Reply:
Removed.